### ISSUE TITLE
Proposal: Allow passing --format multiple times

### ISSUE THREAD

**Issue by jdufresne** [eslint/eslint#14260]
**The version of ESLint you are using.**

v7.22.0

**The problem you want to solve.**

I would like to have ESLint output an HTML file and also output errors to stdout.

I use ESLint on CI, which is configured to publish HTML output. This is a convenient way to look at the errors in a web browser. However, intuitively, many users examine the console output for errors, which is blank. This has led to some confusion. Removing the HTML is not a good option for my team as many users prefer this format, even if other prefer the console output.

Making matters more difficult, there is a single CLI entry point for running all lint tools. This is used by CI but is also used by team members to run tools locally. The single entry point makes it easier to replicate CI results locally. As a result, if the entry point is configured to output results to HTML, then local runs don't show the errors in the console.

**Your take on the correct solution to problem.**

Allow multiple output formatters. For example:

```
$ yarn eslint --format stylish --format html -o eslint/index.html .
```

This command could output HTML to `eslint/index.html` but also output errors to STDOUT using the stylish formatter.

Other lint tools support this feature. For example, [RuboCop](https://docs.rubocop.org/rubocop/index.html) in the Ruby community allows passing multiple formatters.

**Are you willing to submit a pull request to implement this change?**

Maybe, I'm open to it, but I would require a bit of guidance.

**Comment by nzakas** [comment]
This has been brought up several times:

1. https://github.com/eslint/eslint/issues/13244
2. https://github.com/eslint/eslint/issues/8185
3. https://github.com/eslint/eslint/issues/10707

The problem is that ESLint doesn’t implement output formats as reporters, which typically are in control of how they output the results, but rather as formatters, which only receive input and then output a string for the console to report. Without a major rearchitecture of the CLI, or some other hacky solution, this is just not possible.

If you want a workaround, you can always create a custom formatter that does anything you want, including writing to files. Someone has already repackaged all the built in formatters as individual packages, so you can combine them any way you want:

https://github.com/fregante/eslint-formatters

**Comment by kidroca** [comment]
> If you want a workaround, you can always create a custom formatter that does anything you want, including writing to files.
Thanks for the advice 👍 

I needed an html report generated and also to still post terminal output

This is what I ended up with:
https://gist.github.com/kidroca/338ef55ae7aeb6b9f983170d34a27253

**Comment by nzakas** [comment]
Cool, glad you have a solution. A word of advice: grabbing built in formatters inside the ESLint package is not a supported way of doing that and will be going away in the future. I’d recommend using the separately published packages instead.

**Comment by kidroca** [comment]
Unfortunately the `eslint-formatter-html` is not part of `eslint-formatters` and currently is not usable as it renders in Chinese 

![image](https://user-images.githubusercontent.com/12156624/113708223-a1247700-96e9-11eb-9bd9-31c359bfb51f.png)
